js/main-menu.js

```diff
@@ -0,0 +1,129 @@
+(function ($, _) {
+  $(document).ready(function () {
+    amendMarkupOfMenuItemsWithFontAwesomeIcons();
```

Comment:
I would rather call the func `amendMarkupOfMenuItems` and in the description to the `amendMarkupOfMenuItems` func I would add a note that it works with those items that have font awesome icons. Up to you tho.

Reply:
always prefer describe as precise as you can what the function does in the name, rather that in the documentation. If you do the former, then the reader of your code doesn't need to go and look for the function definition to make sure he understands what the function does